rest/Rest.php

```diff
@@ -105,6 +105,28 @@ public static function get() {
 		 */
 		public static function post() {
 
+			$user_id = get_current_user_id();
+			$wishlist_name = isset( $request['wishlist_name'] ) ? $request['wishlist_name'] : '';
```

Comment:
use sanitize_text_field() before you assign to `$wishlist_name`

Reply:
if user is not logged in `$user_id` will be 0. Should check for user id as well.